### PR TITLE
[Playwright] New test fixture for Singular tests

### PR DESCRIPTION
**Pull request by aweng98**
New fixture to reduce duplicated code in Singular tests.

** `npm run build` and `npm run lint` pass without any issue.

Screenshot 2023-03-02 at 3 04 17 PM

** `npx playwright test` ran without any issue.

Screenshot 2023-03-02 at 3 31 41 PM

### CODE REVIEW

File: playwright-e2e/fixtures/singular-fixture.ts [hunk 1]
```diff
@@ -1,20 +1,41 @@
...
     await use(homePage);
+  },
+  dashboardPage: async ({ page }, use) => {
+    const homePage = await launchHomePage(page);
+    await homePage.signUp();
+    await new PreScreeningPage(page).enterInformationAboutYourself();
+    await auth.createAccountWithEmailAlias(page, { email: SINGULAR_USER_EMAIL, password: SINGULAR_USER_PASSWORD });
+    const dashboardPage = new DashboardPage(page);
+    await dashboardPage.waitForReady();
+    await use(dashboardPage);
```
Comment (aweng98) [inline playwright-e2e/fixtures/singular-fixture.ts:38]: New fixture: Load Home page; enter site password; click `Sign Up` button; create a new account with an email alias; wait for Dashboard page ready.

File: playwright-e2e/pages/singular/dashboard-page.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import Table from 'lib/component/table';
 import EnrollMyChildPage from 'pages/singular/enrollment/enroll-my-child-page';
 import { SingularPage } from 'pages/singular/singular-page';
 
-export default class MyDashboardPage extends SingularPage {
+export default class DashboardPage extends SingularPage {
```
Comment (aweng98) [inline playwright-e2e/pages/singular/dashboard-page.ts:8]: Rename to simplify

File: playwright-e2e/playwright.config.ts [hunk 3]
```diff
@@ -65,7 +65,7 @@ const testConfig: PlaywrightTestConfig = {
   use: {
     headless: true,
     /* Maximum time each (browser) action such as `click()` can take. Defaults to 0 (no limit). */
-    actionTimeout: process.env.CI ? 30 * 1000 : 10 * 1000,
+    actionTimeout: process.env.CI ? 20 * 1000 : 15 * 1000,
```
Comment (aweng98) [inline playwright-e2e/playwright.config.ts:68]: to reduce flakyness.

File: playwright-e2e/tests/singular/enrollment/adult-dependent-enrollment.spec.ts [hunk 4]
```diff
@@ -36,19 +31,8 @@ test.describe('Enrol an adult dependent', () => {
...
-    // On “pre-screening” page, answer all questions about yourself with fake values
-    const preScreeningPage = new PreScreeningPage(page);
-    await preScreeningPage.enterInformationAboutYourself();
-
-    // Enter email alias and new password in Login popup
-    await auth.createAccountWithEmailAlias(page, { email: SINGULAR_USER_EMAIL, password: SINGULAR_USER_PASSWORD });
-
-    // On "My Dashboard" page
-    const myDashboardPage = new MyDashboardPage(page);
-    await myDashboardPage.enrollMyAdultDependent();
```
Comment (aweng98) [inline playwright-e2e/tests/singular/enrollment/adult-dependent-enrollment.spec.ts:51]: Delete code are replaced by new fixture `dashboardPage`.

**Review by Rinchan20 — APPROVED** [review]

**Pull request merged**